doc/src/lexing.md

```diff
@@ -8,13 +8,8 @@ starting at index 7 to index 10"). There is also a simple mechanism to
 differentiate lexemes of zero length (e.g. `DEDENT` tokens in Python) from
 lexemes inserted by [error recovery](errorrecovery.md).
 
-Users can write custom lexers that conform to the `lrpar::lex::Lexer` trait.
-This API allows users to deal with streaming data since the parser asks the
-`Lexer` for one token at a time. However, note that users can later ask the
-`Lexer` to return the string from the input matching a lexeme: users need to
-buffer input to provide this information.
-
-Hand-written lexers are not particularly difficult to write and, for better or
-worse, are necessary for many real-world languages. However, a subset of
-languages can use a simpler `lex`/`flex` style approach to lexing, for which
-[`lrlex`](lrlex.html) can be used.
+A a subset of languages can use a simple `lex`/`flex` style approach to lexing,
```

Comment:
Typos `A a`

Reply:
Fixed in https://github.com/softdevteam/grmtools/pull/199/commits/c874d4fc7413eb9b94f3145d1b81fda32ef2168a.